CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing Guidelines
+
+Thanks for thinking about contributing to Project Calico! The success of an
+open source project is entirely down to the efforts of its contributors, so we
+do genuinely want to thank you for contributing.
+
+This document contains some guidance and steps for contributing. Make sure you
+read it thoroughly, because if you miss some of these steps we will have to ask
+you to do them after you make your submission, and that slows down our ability
+to merge it.
+
+## Before Contributing: The Contributor License Agreement
+
+If you plan to contribute in the form of documentation or code, we need you to
+sign our Contributor License Agreement before we can accept your contribution.
+This covers the legalities, confirming that you retain ownership of your
+contributions and that you are licensing them to us.
+
+For some fairly complicated legal reasons we actually have three CLAs.  Which
+you sign depends on whose behalf you are contributing:
```

Comment:
This wording is a bit funny. I think it's missing "Which **one** you sign..." firstly, but even then I don't really like the "whose behalf you are contributing" part.

I think you should change it to something like: "You should sign the CLA that matches who you are making the contribution for."